scene/resources/mesh.h

```diff
@@ -75,6 +75,10 @@ class Mesh : public Resource {
 
 	};
 
+	enum {
+		ARRAY_CUSTOM_COUNT = RenderingServer::ARRAY_CUSTOM_COUNT
```

Comment:
I think it should be ARRAY_CUSTOM_MAX if its the max number of elements, to keep with the rest of the engine terminology.